samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -234,11 +238,24 @@ static const char link_rsrp_usage_str[] =
 	"  -u, --unsubscribe,  Unsubscribe for RSRP info\n";
 
 static const char link_ncellmeas_usage_str[] =
-	"Usage: link ncellmeas --single | --continuous |--cancel\n"
+	"Usage: link ncellmeas --single | --continuous | --cancel | [<search_type>]\n"
 	"Options:\n"
-	"      --single,     Start a neighbor cell measurement and report result\n"
-	"      --continuous, Start continuous neighbor cell measurement mode and report result\n"
-	"      --cancel,     Cancel/Stop neighbor cell measurement if still ongoing\n";
+	"      --single,       Start a neighbor cell measurement and report result\n"
+	"      --continuous,   Start continuous neighbor cell measurement mode and report result\n"
+	"      --cancel,       Cancel/Stop neighbor cell measurement if still ongoing\n"
+	"Search type options, can be used when starting measurements:\n"
+	"  Search type:\n"
+	"                        Default: The modem searches the network it is registered to\n"
+	"                        based on previous cell history. For modem firmware\n"
+	"                        versions < 1.3.1, this is the only valid option.\n"
+	"      --type_ext_light, Search type: extended light.  The modem starts with the same\n"
+	"                        search method as the default type. If needed, it continues to\n"
+	"                        search by measuring the radio conditions and makes assumptions on\n"
+	"                        where networks might be deployed, i.e. a light search.\n"
+	"      --type_ext_comp,  Search type: extended complete.  The modem follows the same\n"
+	"                        procedure as for type_ext_light, but will continue to perform\n"
+	"                        a complete search instead of a light search, and the search is\n"
+	"                        performed for all supported bands.\n";
```

Comment:
I'm seeing now after having written this that TommiK also commented it. Below is still my original comment which we should think about.

The style in which search type is defined doesn't seem to fit well into the usage format (at least couldn't find where we would have used this) although I have no problems understanding it.
I think we have two options for a more normal usage style.

1. `--search_type=ext_light` type of style where values could be `default`, `ext_light`, `ext_comp`.
2. Keep two existing ones (`--type_ext_light` and `--type_ext_comp`) and add `type_default` basically just to have a place where to document the default behavior. Not sure whether this makes sense.

Reply:
After doing TommiK's proposal for usage, I think it is simpler and fine as it is.

Reply:
This is up to you but having an optional parameter which is likely not used most of the time, option #1 would be cleanest from the usage documentation point of view (similarly to, for example, `--wifi_service`).

Reply:
Changed. Please, check. The changes are in a separate commit for now.

Reply:
This is good